### ISSUE TITLE
Misleading translation in English app onboarding

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-documentation#748]
## Avoid duplicates
* [X] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [X] Bug affects both Android **and** iOS, for specific issues / questions that apply only to one operating system, please raise them in the respective repositories:
    * [Android repository](https://github.com/corona-warn-app/cwa-app-android)
    * [iOS repository](https://github.com/corona-warn-app/cwa-app-ios)
* [X] Bug is not already reported in another issue

## Technical details

- Device name: Not important
- iOS/Android version: Not important
- App version: 2.13.1 / 2.13.2

## Describe the bug

In the first screen of the English app onboarding it says:
"You can also **verify** your personal vaccination status." 
Verify means "überprüfen", but not "nachweisen" which is used in the German text. 
It's misleading to say one can "verify" the vaccination status, as one thinks of checking certificates (what CovPassCheck does) when reading the word "verify".
(@MikeMcC399 Would be nice to get your opinion on this, as a native speaker!)

## Steps to reproduce the issue

1. Reset the app
2. Set the language to German
3. Scroll down to the first step of the onboarding
4. Read the text quoted above.

## Expected behaviour

The text should be clear that "nachweisen" and not "überprüfen ("prove" and not "verify") is meant.

## Possible Fix

Change "verify" to "prove".

## Screenshots

| iOS | Android | 
|---|---| 
| <img src="https://user-images.githubusercontent.com/67682506/141698822-01434654-a70f-4fc2-9c47-10171a82f518.png" width="300"> | <img src="https://user-images.githubusercontent.com/67682506/141698827-1d156119-506f-40d7-a209-abf400645a47.jpg" width="300"> |

---
Internal Tracking ID: [EXPOSUREAPP-10569](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10569)

**Comment by MikeMcC399** [comment]
@Ein-Tim 

Your are right that:

```
    <!-- YTXT: onboarding(together) - paragraph -->
    <string name="onboarding_body_5">"You can also verify your personal vaccination status."</string>
```

is not a correct translation of:

```
    <!-- YTXT: onboarding(together) - paragraph -->
    <string name="onboarding_body_5">"Ebenso können Sie Ihren persönlichen Impfstatus nachweisen."</string>
```

The correct verb is "prove" (https://dictionary.cambridge.org/dictionary/english/prove), so an acceptable translation would be:

**"You can also prove your personal vaccination status."**

The word that you suggested (proof) is a noun, so it doesn't exactly fit here.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

> The word that you suggested (proof) is a noun, so it doesn't exactly fit here.

Thanks for your answer! Corrected in the OP.

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 Thanks. Internal Tracking ID: [EXPOSUREAPP-10569](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10569)

**Comment by Ein-Tim** [comment]
PRs fixing this:
- https://github.com/corona-warn-app/cwa-app-android/pull/4392
- https://github.com/corona-warn-app/cwa-app-ios/pull/3846

**Comment by MikeMcC399** [comment]
This issue has been fixed in the CWA Android 2.14.1 release.

## Steps to verify

1. Reset the app
2. Set the language to English (United States)
3. Scroll down in "Let's fight coronavirus together" to the bottom
4. Note the changed text "You can also prove your personal vaccination status."

i.e. the word "verify" has been corrected to read "prove".

**Comment by Ein-Tim** [comment]
Sorry for not coming back to this issue, I'm very busy at the moment. I'll try to verify the fix by todays evening.

**Comment by Ein-Tim** [comment]
I can confirm that this issue has been fixed in the CWA iOS 2.14.1 release.

The Steps to verify the fix are the same as the one [commented](https://github.com/corona-warn-app/cwa-documentation/issues/748#issuecomment-977735083) by @MikeMcC399 above.

Here is a screenshot:

<img src="https://user-images.githubusercontent.com/67682506/143492218-d500746c-2cc8-496a-8fda-42d77157e1ec.png" width="300">

Closing this issue as fixed.
